src/components/listing-page-components/Slider/slider.styles.js

```diff
@@ -0,0 +1,135 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+
+export const SectionContainer = styled.div`
+  background-color: ${colors.alabaster};
+  overflow: hidden;
+`;
+
+export const SliderContainer = styled.div`
+  margin: 0 auto;
+  max-width: 93.75rem;
+
+  .slick-dots {
+    display: none !important;
+    position: relative;
+
+    li {
+      height: 7.5rem;
+      margin: 0;
+      overflow: hidden;
+      width: 9.375rem;
+
+      img {
+        height: 6.25rem;
+        width: 9.375rem;
+      }
+    }
+
+    ${mq.desktop`
+      display: block !important;
+    `}
+  }
+
+  .slick-active {
+    text-align: center;
+  }
+
+  ${mq.desktop`
+    padding: 1.25rem 3.25rem; 
+  `}
+`;
+
+export const ActiveImage = styled.div`
+  max-width: fit-content;
+  opacity: 1;
+  position: relative;
+
+  img {
+    height: 24.8438rem;
+    width: 44.6875rem;
+  }
+`;
+
+export const CustomPaging = styled.div`
+  background-color: rgba(0, 0, 0, 60%);
+  bottom: 0;
+  color: ${colors.white};
+  font-size: 0.75rem;
+  padding: 0.5rem;
+  position: absolute;
+  right: 0;
+
+  ${mq.desktop`
+    display: none;
+  `}
+`;
+
+export const InActiveImage = styled.div`
+  transform: scale(0.9);
+  transition: transform 500ms;
+
+  img {
+    filter: brightness(40%);
+    height: 24.8438rem;
+    margin: 0 auto;
+    width: 44.6875rem;
+  }
+`;
+
+export const Next = styled.div`
+  background-color: ${colors.mineShaft};
+  border-radius: 50%;
+  cursor: pointer;
+  height: 1.875rem;
+  opacity: 0.6;
+  position: absolute;
+  right: 0%;
+  top: 50%;
+  width: 1.875rem;
+  z-index: 3;
+
+  svg {
+    transform: scale(0.5) translate(-50%, -50%);
+  }
+
+  ${mq.desktop`
+    height: 3.75rem;
+    transform: translate(1.875rem, -100%);
+    width: 3.75rem;
+
+    svg {
+      transform: scale(1);
+    }
+    
+  `}
+`;
+
+export const Previous = styled.div`
```

Comment:
dedupe into one please.
```suggestion
export const Arrow = styled.div`
```